templates/web/guides/models/defining.md.erb

```diff
@@ -242,10 +305,42 @@ module NVM
     include Origen::Model
 
     def initialize(options={})
+      @size_in_kB = 128
     end
   end
 end
 
+# lib/nvm/memory_64_b954.rb
+module NVM
+  class MEMORY_64_B954
+    include Origen::Model
+
+    def initialize(options={})
+      @size_in_kB = 64
+    end
+  end
+end
+
+# lib/nvm/memory_32_b954.rb
+module NVM
+  class MEMORY_32_B954
+    include Origen::Model
+
+    def initialize(options={})
+      @size_in_kB = 32
+    end
+  end
+end
+
+# lib/memories.rb
+class MEMORIES_B294 < ::Array
+
+  def <<(sub_block)   # to meet sub_block_group requirement
```

Comment:
As above, would either remove this, or change it to a different example that uses a custom container that doesn't inherit from `Array` and therefore does require this to be defined.

Reply:
Yeah the reason I said that `<<` was the requirement initially is that the `sub_block_group` method explicitly uses it.   Will try to clarify that.